presentation/src/main/java/com/anytypeio/anytype/presentation/auth/account/MigrationHelperDelegate.kt

```diff
@@ -26,8 +26,9 @@ interface MigrationHelperDelegate {
                 .map { result ->
                     when(result) {
                         is Resultat.Failure -> {
-                            if (result.exception is MigrationFailedException.NotEnoughSpace) {
-                                State.Failed.NotEnoughSpace
+                            val exception = result.exception
+                            if (exception is MigrationFailedException.NotEnoughSpace) {
```

Comment:
Might need to convert the number to MB at this step, waiting for confirmation from the Go team.